### ISSUE TITLE
Picker not openable

### ISSUE THREAD

**Issue by gorbypark** [andrew-levy/swiftui-react-native#38]
Just playing around and noticed that the picker component doesn't seem to actual open!  I have a basic screen and when pressing on the picker, nothing happens.  I added in a button just to make sure that there wasn't something weird going on and the entire screen wasn't registering presses or something like that (the button shows the alert completely fine).  I also tried taking out the View wrapper and making it a VStack in case that had something to do with it, but I got the same results.  Setting the useBinding value to "Option 2" or "Option 3" makes the component show that value, so it seems that the binding is working.

```tsx
import { View } from "react-native";
import { Picker, useBinding, Text, Button } from "swiftui-react-native";
import { Alert } from "react-native";

export default function Translate() {
  const selection = useBinding("Option 1");

  return (
    <View style={{ flex: 1, gap: 20 }}>
      <Button action={() => Alert.alert("hello")} title="does this work?" />
      <Picker selection={selection}>
        <Text>Option 1</Text>
        <Text>Option 2</Text>
        <Text>Option 3</Text>
      </Picker>
    </View>
  );
}
```

**Comment by gorbypark** [comment]
After some trial and error, it seems that setting the frame height to at least 8 makes it work.  7 and below and it doesn't open.
```jsx
      <Picker selection={selection} frame={{ height: 8 }}>
        <Text>Option 1</Text>
        <Text>Option 2</Text>
        <Text>Option 3</Text>
      </Picker>
      ```

**Comment by andrew-levy** [comment]
Interesting! By default, the picker should have a height of 35, so I wouldn't expect this behavior. Let me look into it to see if there's a better way to fix this. Thanks for testing this out! @gorbypark

**Comment by andrew-levy** [comment]
Ah okay there was a bug. I will push this fix out in the next minor release with a few other changes. Will keep you posted!

**Comment by andrew-levy** [comment]
@gorbypark This should be fixed in swiftui-react-native@6.1.0. Let me know if you're still seeing the issue!
